File: 단계별로_풀어보기/6. 심화 1단계/2_3003.js
// 입력
const fs = require('fs')
const inputArr = fs.readFileSync(0, 'utf-8').toString().trim().split('\n')

// 입력 파싱
const numArr = inputArr[0].trim().split(' ').map(Number)

// 비교군 배열
const targetArr = [1, 1, 2, 2, 2, 8]

// 출력할 string 에 값 추가
let result = ''

targetArr.forEach((target, idx) => {
  const temp = target - numArr[idx]
  result += temp + ' '
})

process.stdout.write(result)

File: 단계별로_풀어보기/6. 심화 1단계/2_3003.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

inputArr = list(map(int, input().strip().split()))

targetArr = [1, 1, 2, 2, 2, 8]

resultArr = [(targetArr[i] - inputArr[i]) for i in range(6)]

output(f"{' '.join(str(res) for res in resultArr)}")